doc/source/performance/threadpool.rst

```diff
@@ -4,41 +4,44 @@
+A dynamic thread pool enables the server to maintain top performance with a large number of clients. A thread pool decreases the number of threads on the server, which also reduces the context switching and hot locks contentions. A thread pool has the most effect with ``OLTP`` workloads (relatively short CPU-bound queries).
 
-In order to enable the thread pool variable :variable:`thread_handling` should be set up to ``pool-of-threads`` value. This can be done by adding: ::
+In order to enable the thread pool, the variable :variable:`thread_handling` should be set up to ``pool-of-threads`` value. This can be done by adding this setting to the |MySQL| configuration file :file:`my.cnf`: ::
 
  thread_handling=pool-of-threads
 
-to the |MySQL| configuration file :file:`my.cnf`.
+.. seealso::
+
+   |MySQL| Documentation
+   `thread-handling <https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_thread_handling>`_
 
-Although the default values for the thread pool should provide good performance, additional `tuning <https://kb.askmonty.org/en/threadpool-in-55/#optimizing-server-variables-on-unix>`_ can be performed with the dynamic system variables described below.
+Although the default values for the thread pool should provide good performance, additional tuning can be performed with the dynamic system variables described in system-variables_.
 
 .. note::
  
-  Current implementation of the thread pool is built in the server, unlike the upstream version which is implemented as a plugin. Another significant implementation difference is that this implementation doesn't try to minimize the number of concurrent transactions like the ``MySQL Enterprise Threadpool``. Because of these things this implementation isn't compatible with the upstream one.
+ Unlike the upstream version, which is installed as a plugin, the current thread pool operation is built in the server. Another significant difference is that this thread pool version does not attempt to minimize the number of concurrent transactions like the ``MySQL Enterprise Threadpool``. Due to these differences, this implementation is not compatible with the upstream one.
 
 Priority connection scheduling
 ==============================
 
-Even though thread pool puts a limit on the number of concurrently running queries, the number of open transactions may remain high, because connections with already started transactions are put to the end of the queue. Higher number of open transactions has a number of implications on the currently running queries. To improve the performance new :variable:`thread_pool_high_prio_tickets` variable has been introduced.
+Though the thread pool limits the number of concurrently running queries, the number of open transactions may remain high; because the queue puts connections with already started transactions at the end. A high number of open transactions has several implications on the currently running queries.
```

Comment:
```suggestion
Though the thread pool limits the number of concurrently running queries, the number of open transactions may remain high. This is because the queue puts connections with already started transactions at the end. A high number of open transactions has several implications on the currently running queries.
```

Reply:
Rewritten.